textile/features.textile

```diff
@@ -715,7 +715,7 @@ h3(#realtime-presence). RealtimePresence
 ** @(RTP8c)@ A @PRESENCE ProtocolMessage@ with a @PresenceMessage@ with the action @ENTER@ is sent to the Ably service. The @clientId@ attribute of the @PresenceMessage@ must not be present. Entering without an explicit @PresenceMessage#clientId@, implicitly uses the @clientId@ for the current connection
 ** @(RTP8d)@ Implicitly attaches the @RealtimeChannel@ if the channel is in the @INITIALIZED@ state. However, if the channel is in the @DETACHED@ or @FAILED@ state, the @enter@ request results in an error
 ** @(RTP8e)@ Optional data and/or extras can be included when entering a channel that will be encoded and decoded as with normal messages. A test should exist to ensure data and extras used with enter are encoded & decoded correctly. Also, when data and/or extras is provided when entering, but neither data nor extras are provided when leaving, the data attribute should be emitted in the @LEAVE@ event for this client
-** @(RTP8f)@ If the client library is authenticated but unidentified (i.e. @clientId@ is a wildcard @'*'@ or client is anonymous), the @enter@ request results in an error immediately
+** @(RTP8f)@ If the connection is in the @CONNECTED@ state, and the @RealtimeClient#clientId@ value is either @'*'@ (a wildcard value indicating that this client is permitted to associate any client identifier it wishes with the operations it performs) or @null@ (indicating that the client is anonymous and is not permitted to associate a client identifier with the operations it performs), then the @enter@ request results in an error immediately.
```

Comment:
I would prefer two sentences instead of one long with embraced explanations. Or provide spec points for explanations (create them if they're not exist?)

Reply:
I've changed this to a list instead, so hopefully it reads more clearly now.